quiz.js

```diff
@@ -1,21 +1,42 @@
+var charBox= document.getElementById("character");
+var heightBox=document.getElementById("height");
+var growBtn= document.getElementById("grow");
 
 
+growBtn.addEventListener("click", function(){
+	tree(treeObject);
+});
+heightBox.addEventListener("keyup", function(){
+	treeObject.height = parseInt(heightBox.value) + 1;
+	if(event.code==="Enter"){
+		tree(treeObject);
+	}
+});
+charBox.addEventListener("keyup", function(){
+	treeObject.character = charBox.value;
+	if(event.code==="Enter"){
+		tree(treeObject);
+	}
+});
 
 
 var treeObject = {
-	height: null,
-	character: null
+	height: "",
+	character: ""
 };
 
 
 function tree(treeObj){
+	if(treeObject.height === "" || treeObject.character === ""){
+		console.log("Both fields must have a value");
```

Comment:
Change this from a console log into an alert (per the requirement) and you're good to go! Nice, simple, and super concise!

Though the spacing / indentation on lines 34-47 are also wonky. (Proper indention is important if you intend to work with tools like Jade or languages like Python)